.gitignore

```diff
@@ -2,4 +2,7 @@
 .vscode/
 
 #Ignore all the libraries (some time ago it was not ignored and created a big mess)
-assets/libs/
\ No newline at end of file
+assets/libs/
+
+#Ignore tests folder
```

Comment:
As I think about it now, this specific change should not really be there, but I thought that since a lot of changes has been made in the .gitignore file lately, it would not be horrible to include this commit in this pull request